crates/polars-core/src/series/implementations/duration.rs

```diff
@@ -407,6 +407,13 @@ impl SeriesTrait for SeriesWrap<DurationChunked> {
     fn min_as_series(&self) -> Series {
         self.0.min_as_series().into_duration(self.0.time_unit())
     }
+    fn std_as_series(&self, ddof: u8) -> Series {
+        self.0.std_as_series(ddof).into_duration(self.0.time_unit())
```

Comment:
So here you must cast the result back to integers and then call `into_duration`.

Reply:
Done! :)